community/codestar/custom-ci-cd-pipeline/README.md

```diff
@@ -0,0 +1,48 @@
+# custom-ci-cd-pipeline
+
+This template creates a custom CodePipeline pipeline for continuous integration and continuous delivery, for *two environments* named `staging` and `production`.
+
+## Steps
+
+1. Source: fetches the latest version of a branch from a CodeCommit repository.
+2. Build staging: builds the project using CodeBuild by executing the `buildspec.yml` file.
+3. Staging deployment: deploys the output of step 2 using CodeBuild by executing the `deployspec.yml` file.
+4. Manual approval.
+5. Build production: builds the project using CodeBuild by executing the `buildspec.yml` file.
+6. Staging deployment: deploys the output of step 5 using CodeBuild by executing the `deployspec.yml` file.
+
+## CodeCommit repository
+The CodeCommit repository must exists in the same region where you are creating the pipeline and the name it is provided as a parameter to the stack.
```

Comment:
@EgidioCaprino I would also include something about how the CodeCommit repository needs to be created prior to running this CloudFormation. I think this would address @cmmeyer's concern.

Reply:
Alright, I've updated it. What do you think @ialford ?

Reply:
Looks good to me 👍